Source/UIViewController+STZPopupView.swift

```diff
@@ -67,7 +67,7 @@ extension UIViewController {
     - parameter popupView: Popup view
     - parameter config:    Config (Option)
     */
-    public func presentPopupView(popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig()) {
+    public func presentPopupView(_ popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig()) {
```

Comment:
I prefer not to use underscore. 
`public func present(popupView: UIView, config: STZPopupViewConfig = STZPopupViewConfig())`

See https://developer.apple.com/reference/uikit/uiviewcontroller/1621380-present

API interface will change. It cannot be helped...